6 configure tailwind variables

HelChris opened the pull request:
Add variables according to mobile MVP design for: 

- headings and text sizes
- colors and bgcolors
- fonts from google fonts

HelChris committed:
style: add CSS variables in theme

HelChris committed:
style: add fonts and colors

HelChris committed:
style: add google font link

HelChris committed:
fix: correct curlybraces around @themes

HelChris committed:
style: add rem and rename variables

HelChris committed:
style: add bgcolor variable on body

Anclagen commented on src/styles/index.css:
```diff
@@ -60,13 +135,10 @@ button:focus-visible {
-    background-color: #f9f9f9;
+    color: #747bff; /* update when design is ready*/
   }
 }
```
I'm not 100% sure but I think this might conflict with our custom toggle for the colour scheme and override to system preferences. I think we need to set it up like so
```css
:root {
  line-height: 1.5;
  font-weight: 400;
  color-scheme: light;
  color: var(--color-primary-nightwing-hex);
  background-color: var(--color-accent-eggshell-hex);
  font-synthesis: none;
  text-rendering: optimizeLegibility;
  -webkit-font-smoothing: antialiased;
}

.dark {
  color-scheme: dark;
  color: var(--color-accent-eggshell-hex);
  background-color: var(--color-primary-nightwing-hex);
}
```

Anclagen reviewed (changes requested):
Looks good, just one problem to rectify with using a manual toggle rather than defaulting to system preference for theme.

HelChris committed:
style: change from prefers-color-scheme to class selector

HelChris commented:
added .light class, removed prefers-color-scheme. (.light because designers MVP). If that is wrong and the default need to be light and the class to be .dark then I'll change it again.

Anclagen commented:
Mia’s code uses the `.dark` class on the body, which aligns with Tailwind’s `dark:` variant implementation. Tailwind treats light as the default theme.

HelChris committed:
style: add correct light-/darkmode style

HelChris committed:
style: add correct bg-color

HelChris commented:
ah, ok. Fixed now.
